flink-cdc-common/src/main/java/com/ververica/cdc/common/schema/Schema.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.schema;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+import com.ververica.cdc.common.types.DataField;
+import com.ververica.cdc.common.types.DataType;
+import com.ververica.cdc.common.types.DataTypeRoot;
+import com.ververica.cdc.common.types.DataTypes;
+import com.ververica.cdc.common.types.RowType;
+import com.ververica.cdc.common.utils.Preconditions;
+
+import javax.annotation.Nullable;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.ververica.cdc.common.types.DataTypes.ROW;
+
+/** Schema of a table or data collection. */
+@PublicEvolving
+public class Schema implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private final List<Column> columns;
+
+    private final List<String> primaryKeys;
+
+    private final Map<String, String> options;
```

Comment:
What will be stored in the `options`? It is strange that there is a `options` in `Schema`.

Reply:
I think this option for store like `Charset`  name ?

Reply:
Each table may have its own chunkKey, its own chunkSize